include/SFML/Graphics/Texture.hpp

```diff
@@ -35,10 +35,11 @@
 
 namespace sf
 {
-class Window;
+class InputStream;
 class RenderTarget;
 class RenderTexture;
-class InputStream;
+class Text;
+class Window;
```

Comment:
No need to reorder the declarations here... Add a single line instead of modifying 3.

Reply:
I though having them alphabetically made more sense then random.

Reply:
👍